docs/modules/clients/pages/java.adoc

```diff
@@ -24,14 +69,53 @@ which you may already have done to start using Hazelcast:
 </dependency>
 ----
 
-You can find Hazelcast Java client's code samples https://github.com/hazelcast/hazelcast-code-samples/tree/master/clients[here^].
+--
+====
+
+You can find {java-client} code samples in the https://github.com/hazelcast/hazelcast-code-samples/tree/master/clients[Hazelcast Code Samples repository].
+
+TIP: For a tutorial on getting started with Java in an embedded topology, see xref:getting-started:get-started-java.adoc[].
+
+=== Get started with {java-client-new} (BETA)
+
+To get started using the {java-client-new}, you need to include the `hazelcast-java-client-{full-version}.jar` dependency in your classpath. You can then start using this client as if
+you are using the Hazelcast API. 
+
+If you prefer to use Maven, add the `hazelcast-java-client` dependency to your `pom.xml`:
+
+// check conversation about `hazelcast-enterprise-java-client` dependency and note i.e. is it as below?
+// If you prefer to use Maven, add either the `hazelcast-java-client` dependency to your `pom.xml`, or the `hazelcast-enterprise-java-client` dependency (if you have an {enterprise-product-name} license).
+
+
+[source,xml,subs="attributes+"]
+----
+<dependency>
+    <groupId>com.hazelcast</groupId>
+    <artifactId>hazelcast-java-client</artifactId>
+    <version>{full-version}</version>
+</dependency>
+----
+
+To migrate from the {java-client} to the {java-client-new}, you only have to update the dependency as described above.
```

Comment:
In general, this is true, but do we want to mention possible problems that can happen ? For example, missing APIs due to missing features or missing APIs due to throwing UnsupportedOperationException (and removed therefore) will cause compile problems.

Also, the declerative config and system properties will not work.

Reply:
They're all covered right below in https://deploy-preview-1289--hardcore-allen-f5257d.netlify.app/hazelcast/5.5/clients/java#java-client-standalone-beta section